app/controllers/tribes_controller.rb

```diff
@@ -24,37 +26,9 @@ def destroy
 
   def auto_suggest
     contractor_opts = format_params_for_auto_suggest
-    ga = GeneticAlgorithm.new(dev_opts(contractor_opts))
-    ga.evolve(25)
-    @pending_tribe = PendingTribe.new(ga.fittest_chromosome.dna.map{|dev| dev.id})
+    fittest = GeneticAlgorithm.new(dev_opts(contractor_opts)).evolve(25)
+    @pending_tribe = PendingTribe.new(fittest.dna.map{|dev| dev.id})
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:id` as an argument to `map` instead of a block.